psl-evaluation/src/main/java/org/linqs/psl/utils/evaluation/statistics/DiscretePredictionComparator.java

```diff
@@ -110,6 +110,7 @@ public DiscretePredictionStatistics compare(Predicate p, int maxBaseAtoms) {
 		}
 		
 		tn = maxBaseAtoms - tp - fp - fn;
+        //System.out.println("TEST:\ntp " + tp + " fp " + fp + " tn " + tn + " fn " + fn);
```

Comment:
Use tabs instead of spaces (I personally hate this, but the rest of the codebase uses this).